<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProntoSoccorso</class>
 <widget class="QWidget" name="ProntoSoccorso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 235, 239);</string>
  </property>
  <widget class="QLabel" name="InformazioniProfilo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>581</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 235, 239,0);
color: rgb(230, 235, 239);</string>
   </property>
   <property name="text">
    <string>Info paziente</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>55</y>
     <width>290</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(230, 235, 239,0.6);
background-color: rgba(0, 57, 97,0);</string>
   </property>
   <property name="text">
    <string>Dati personali </string>
   </property>
  </widget>
  <widget class="QLabel" name="Sheda">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>290</width>
     <height>210</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 235, 239,0);
border: 1px solid rgba(230, 235, 239,0.6);
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);

border-radius: 6px;
   

</string>
   </property>
   <property name="text">
    <string>Data di nascita :</string>
   </property>
  </widget>
  <widget class="QLabel" name="LogoPazienti">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>135</y>
     <width>160</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 235, 239,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>MATERIALE/IconaPazientiom.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>450</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>E:/UI UFFICIALI/MATERIALE/Back.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Nome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Cognome">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="DataDiNascita">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CodiceFiscale">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Sesso">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CittaDiNascita">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Cellulare">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Residenza">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Sheda_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>315</y>
     <width>290</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 235, 239,0);
border: 1px solid rgba(230, 235, 239,0.6);
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>290</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(230, 235, 239,0.6);
background-color: rgba(0, 57, 97,0);</string>
   </property>
   <property name="text">
    <string>Dati paziente</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);

border-radius: 6px;
   

</string>
   </property>
   <property name="text">
    <string>Stato ricovero:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);

border-radius: 6px;
   

</string>
   </property>
   <property name="text">
    <string>Prenotazione:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Ricovero">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonElimina">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgba(230, 235, 239,0.6);
background-color: rgba(0, 57, 97,0);
border-radius: 10px;
}
QPushButton:hover {
background-color: rgba(40, 40, 40,0.5);
}
</string>
   </property>
   <property name="text">
    <string>Elimina</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonModifica">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(0, 57, 97,0);
	color: rgb(230, 235, 239);
color: rgb(230, 235, 239);
border-radius: 10px;
}
QPushButton:hover {
background-color: rgba(40, 40, 40,0.5);
    }</string>
   </property>
   <property name="text">
    <string>Modifica</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="Prenotazione">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);

border-radius: 6px;
   

</string>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>Back</zorder>
  <zorder>Sheda</zorder>
  <zorder>InformazioniProfilo</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>LogoPazienti</zorder>
  <zorder>Nome</zorder>
  <zorder>Cognome</zorder>
  <zorder>DataDiNascita</zorder>
  <zorder>CodiceFiscale</zorder>
  <zorder>Sesso</zorder>
  <zorder>CittaDiNascita</zorder>
  <zorder>Cellulare</zorder>
  <zorder>Residenza</zorder>
  <zorder>Sheda_2</zorder>
  <zorder>label_2</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>Ricovero</zorder>
  <zorder>ButtonElimina</zorder>
  <zorder>ButtonModifica</zorder>
  <zorder>Prenotazione</zorder>
 </widget>
 <resources>
  <include location="../asa.qrc"/>
 </resources>
 <connections/>
</ui>
